test/e2e/client.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"k8s.io/client-go/kubernetes"
+	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/vmware-tanzu/velero/pkg/client"
+)
+
+// testClient contains different API clients that are in use throughout
+// the e2e tests.
+type testClient struct {
```

Comment:
Ah, sorry I didn't include this in the last review! Is this also part of the follow up refactor branch? I can't see it being used here yet.

Reply:
Yep. Note to self: don't do things in a hurry. Or maybe don't do anything today. Argh!